<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inbody_Dialog</class>
 <widget class="QDialog" name="inbody_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>30</y>
     <width>62</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>체 중</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>키</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>골격근량</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>B M I</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>체지방률</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>복부지방률</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>정보입력</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>25</y>
     <width>101</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="user_id">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_2">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_3">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_4">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_5">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_6">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>25</y>
     <width>101</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="user_id_7">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_8">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_9">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_10">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_11">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_id_12">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>70</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>사용자 ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>세포 내 수분</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>세포 외 수분</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>단 백 질</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>무 기 질</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="text">
       <string>체 지 방</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>80</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>인바디 점수 : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inbody_lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
